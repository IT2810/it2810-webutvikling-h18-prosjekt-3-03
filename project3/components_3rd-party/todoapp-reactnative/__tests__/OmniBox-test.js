import React from 'react';
import renderer from 'react-test-renderer';
import OmniBox from '../OmniBox';

it('renders correctly', () => {
    // TODO: test shallowly to avoid having to recreate snapshot if subcomponents change
    const tree = renderer
        .create(
            <OmniBox
                value={"Test value"}
            />)
        .toJSON();
    expect(tree).toMatchSnapshot();
});
