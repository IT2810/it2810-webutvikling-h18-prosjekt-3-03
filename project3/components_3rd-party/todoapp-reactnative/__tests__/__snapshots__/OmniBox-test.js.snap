// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<TextInput
  allowFontScaling={true}
  blurOnSubmit={false}
  onChange={[Function]}
  onSubmitEditing={[Function]}
  placeholder="Add a todo or Search"
  style={
    Object {
      "backgroundColor": "#fff",
      "borderColor": "#eee",
      "borderRadius": 8,
      "borderWidth": 1,
      "fontSize": 16,
      "height": 36,
      "marginBottom": 0,
      "padding": 4,
    }
  }
  underlineColorAndroid="transparent"
  value=""
/>
`;
